core/modules/file/file.file.inc

```diff
@@ -162,8 +162,8 @@ function file_file_mimetype_mapping_alter(&$mapping) {
  */
 function file_file_load($files) {
   // Add alt and title text to images.
-  $alt = variable_get('file_entity_alt', '[file:field_file_image_alt_text]');
-  $title = variable_get('file_entity_title', '[file:field_file_image_title_text]');
+  $alt = variable_get('file_entity_alt', '[file:field_fiat]');
```

Comment:
Not sure I like "magic" short names here. As a coder that has no context, I would like to be able to tell by the name instead of having to "decipher" it.

Reply:
My preference is also for the verbose token name.